test/liblll/EndToEndTest.cpp

```diff
@@ -583,24 +586,34 @@ BOOST_AUTO_TEST_CASE(allgas)
 
 BOOST_AUTO_TEST_CASE(send_two_args)
 {
-	char const* sourceCode = R"(
-		(returnlll
-			(send 0xdead 42))
-	)";
-	compileAndRun(sourceCode);
-	callFallbackWithValue(42);
-	BOOST_CHECK(balanceAt(Address(0xdead)) == 42);
+	// "send" does not retain enough gas to be able to pay for account creation.
+	// Disabling for non-tangerineWhistle VMs.
+	if (dev::test::Options::get().evmVersion().canOverchargeGasForCall())
```

Comment:
Hm, can we output a boost message that these are skipped due to the vm?

Also, how about?
```
if (..)
  return;
```

Reply:
I tried return, but the macro seems to create a function that returns something.

Reply:
I'll look into telling boost to skip a test, it seems to have such a feature.

Reply:
Changed it accordingly. This will probably show up in the xml test report.